sdk/ml/azure-ai-ml/tests/component/e2etests/test_component.py

```diff
@@ -526,6 +526,9 @@ def test_tensorflow_component(
         tensorflow_component_resource = client.components.create_or_update(component_entity)
         assert tensorflow_component_resource.distribution.__dict__ == tensorflow_distribution(has_strs=True)
 
+    @pytest.mark.skip(
+        "Could not rerecord the test , errors: (UserError) Failed to update component test_81585734883"
```

Comment:
Who is on point to fix these? Pipeline SDK team? (Can we create bugs to track all these tests that we skip, or ideally, we get these fixed first?)

Reply:
We follow up with work items with the reponsible teams. I will do the same with STCA.  These are the recorded test and should be recordable (the way I did for 5 other), else it goes to the owners. In VSO we used to put the workitem linked in the PR, but since this is public we don't put that details now...

Reply:
Yes, let's not publish the ADO items but thanks for following up, let's be sure to do that.